app/src/main/res/layout/fragment_mentor.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/main"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <Button
+        android:id="@+id/fav_button"
+        android:layout_width="117dp"
+        android:layout_height="34dp"
+        android:layout_marginTop="52dp"
+        android:layout_marginEnd="27dp"
+        android:background="@drawable/favorite_project_button"
+        android:text="@string/fav_button_text"
+        android:textColor="#222B5C"
+        android:textSize="15dp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
```

Comment:
Can you add `android:fontFamily="@font/montserrat_bold"` to fit with Figma? I may have messed up the string, but Android studio should autocomplete for you.